pyanaconda/ui/common.py

```diff
@@ -694,7 +694,10 @@ def collect_spokes(mask_paths, category):
     spokes = []
     for mask, path in mask_paths:
         spokes.extend(collect(mask, path,
-                      lambda obj: hasattr(obj, "category") and obj.category is not None and obj.category.__name__ == category))
+                      lambda obj: hasattr(obj, "category")
+                                  and obj.category is not None
+                                  and hasattr(obj.category,"__name__")
+                                  and obj.category.__name__ == category))
```

Comment:
The last two lines can be reduced to ``getattr(obj.category, "__name__", None) == category``.